agent/testagent.go

```diff
@@ -314,6 +314,7 @@ func TestConfig() *Config {
 	cfg.Datacenter = "dc1"
 	cfg.Bootstrap = true
 	cfg.Server = true
+	cfg.CheckEnableExec = true
```

Comment:
why? can't we just add it to the handful of tests that need it? It is the new default after all. We should see what breaks.